libs/configuration-variables-resolver/tests/UnifiedConfigurationResolverTest.php

```diff
@@ -0,0 +1,249 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ConfigurationVariablesResolver\Tests;
+
+use Keboola\ConfigurationVariablesResolver\SharedCodeResolver;
+use Keboola\ConfigurationVariablesResolver\UnifiedConfigurationResolver;
+use Keboola\ConfigurationVariablesResolver\VariablesResolver;
+use Keboola\StorageApi\BranchAwareClient;
+use Keboola\StorageApiBranch\ClientWrapper;
+use Keboola\VaultApiClient\Variables\Model\Variable;
+use Keboola\VaultApiClient\Variables\VariablesApiClient;
+use PHPUnit\Framework\TestCase;
+use Psr\Log\NullLogger;
+
+class UnifiedConfigurationResolverTest extends TestCase
```

Comment:
Hele, tohle neni vubec spatny, ale prijde mi to takovy pomixovany:
* `UnifiedConfigurationResolverTest` bych cekal, ze bude testovat ten resolver, coz v pripade soucasny implementace je v podstate jen otestovat pres mocky, ze to preda spravne parametry do variables a shared-code resolveru
* naopak v tomhle testu, kdyz bys misto `createResolver` pouzil primo tu factory, co si tu udelal, tam as super polo-funkcni test na to, ze ta factory vytvori spravne nakonfigurovany resolvery (tim, ze ty resolvery udelaji ty spravny API requesty)
* je potreba testovat vsechny ty varianty (variable/data override)? Protoze to uz je funkcionalita primo v tom variables resolveru